services/consensusalgo/benchmarkconsensus/common.go

```diff
@@ -45,10 +46,12 @@ func (s *service) requiredQuorumSize() int {
 }
 
 func (s *service) saveToBlockStorage(ctx context.Context, blockPair *protocol.BlockPairContainer) error {
+	logger := s.logger.WithTags(trace.LogFieldFrom(ctx))
```

Comment:
do we have to create a new trace every func ?

Reply:
no, but we have to pass the trace to the logger somehow